services/i18n.js

```diff
@@ -0,0 +1,94 @@
+const has = require('lodash/has');
+const get = require('lodash/get');
+
+const yaml = require('yamljs');
+const es = yaml.load('./locales/es.yml');
+const en = yaml.load('./locales/en.yml');
+
+// default language
+let defaultLanguage = 'en';
+let language = defaultLanguage;
+const translations = Object.assign(en, es);
+
+/**
+* guess language setting based on http headers
+*/
+const guessLanguage = (request) => {
+
```

Comment:
Rather than developing all this stuff by ourselves. I suggest using an established library like this one https://github.com/jshttp/accepts. This library is also utilised by `express` and  `koa`.